pkgs/development/libraries/tidyp/default.nix

```diff
@@ -1,10 +1,11 @@
 { lib, stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
-  name = "tidyp-1.04";
+  pname = "tidyp";
+  version = "1.04";
 
   src = fetchurl {
```

Comment:
This should use fetchFromGitHub.

Reply:
not sure if we really want to keep this: https://github.com/petdance/tidyp
it is a fork of: https://github.com/htacg/tidy-html5

Reply:
already exists in packages: https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/tools/text/html-tidy/default.nix
not sure what todo about it